docs/tutorials/training_transformers.md

```diff
@@ -0,0 +1,93 @@
+# Training Transformers
+
+This tutorial shows how to train a transformer policy network.
+
+<div class="admonition note">
+<p class="admonition-title">Note: Understand how to launch training runs and view results first!</p>
+
+Before trying to train a transformer, it might be useful to read the following tutorials:
+- [how to launch training runs](./configs.html)
+- [how to view training results](./viewing_results.html)
+- [how to launch multiple training runs efficiently](./hyperparam_scan.html) 
+
+</div>
+
+A template with tuned parameters for transformer based policy networks is defined in `robomimic/config/default_templates/bc_transformer.json`.
+
+#### 1. Using default configurations
+
+The easiest way to train a transformer policy network is to pass the default template json to the main training script `train.py` via the `--config` argument. The dataset can be specified by setting the `data` attribute of the `train` section of the config json, or specified via the `--dataset` argument.
```

Comment:
I would also add a small note saying that you may need to change other settings such as the rollout horizon, depending on your dataset and simulation environment. This can also motivate looking at (2)